src/main/java/org/kitchen/Kitchen.java

```diff
@@ -11,33 +11,44 @@
     PIZZA_QUEMADA
 }
 
+enum Ingredient {
+    ELOTE,
+    PINA,
+    ATUN
+}
+
 public class Kitchen {
 
-    Flavor flavor = null;
+    Flavor flavor;
+    Ingredient ingredients;
 
     public String cookingInstruction(Instruction instruction) {
 
-        String ingredient = instruction.getIngredient();
+        try{
+            ingredients = Ingredient.valueOf(instruction.getIngredient());
```

Comment:
Do this validations need to be done here? Or it-s better to validate at the input?